data/ui/preferences-window.ui

```diff
@@ -818,7 +818,7 @@ SPDX-License-Identifier: GPL-2.0-or-later
     <section>
       <item>
         <!-- TRANSLATORS: Open a dialog to connect to an IP or Bluez device -->
-        <attribute name="label" translatable="yes">Connect to…</attribute>
+        <attribute name="label" translatable="yes">Add device by IP...</attribute>
```

Comment:
Actually, _one_ last tweak. GNOME Typography wants us to use a _real_ ellipsis character, rather than three periods.

```suggestion
        <attribute name="label" translatable="yes">Add device by IP…</attribute>
```

Reply:
(I've applied that change, and set this to merge as soon as the CI passes. Thanks again @realAbhi-web!)